docs/core/apps.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Build apps with .NET
+description: .NET can be used for building many kinds of apps.
+ms.date: 1/3/2024
+ms.custom: "updateeachrelease"
+---
+
+# Build apps with .NET
+
+.NET has support for building many kinds of apps, including client, cloud, and gaming.
```

Comment:
This is incomplete duplicate of "Application Models" from https://dotnet.microsoft.com/en-us/learn/dotnet/what-is-dotnet

Reply:
I updated the list to make it better match the website (which I agree is better).